packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +541,34 @@ void main() {
           ));
           handle.dispose();
         });
+
+        testWidgets('Still builds when shrinkWrap, primary, and physics properties are set', (WidgetTester tester) async {
```

Comment:
Could you also add a few more tests that ensure that the `shrinkWrap`, `primary`, and `physics` properties are being used by the application? 

For example, if `shrinkWrap` was true/false, check to see if the expected behavior is observed. Same thing for `primary` and `physics` properties. Also, add tests to make sure that the default behavior doesn't change from the current behavior.

Reply:
I added additional tests for physics to check for behavior changes. I removed `shrinkWrap` and `primary` so this PR just adds the `physics` property.